userspace/libsinsp/cgroup_limits.cpp

```diff
@@ -55,6 +55,11 @@ bool read_cgroup_list_count(const std::string& subsys,
  	std::string cpuset_cpus((std::istreambuf_iterator<char>(cg_val)),
 			    std::istreambuf_iterator<char>());
 
+	if(cpuset_cpus.empty())
```

Comment:
I was curious what the behavior would be if the `ifstream` was in a bad state after opening the file.  I hoped the resulting `cpuset_cpus` would be empty.  I did a little experiment and I ended up with a "1" (I know not why).

I suggest that you check the state of `cg_val` after construction, just to be extra careful:

```c++
std::ifstream cg_val(path);

if(!cg_val)
{
        return false;
}
```